temporal-sdk/src/test/java/io/temporal/workflow/ActivityApplicationFailureRetryTest.java

```diff
@@ -46,12 +49,12 @@
 
   @Test
   public void testActivityApplicationFailureRetry() {
-    WorkflowTest.TestWorkflow1 workflowStub =
+    TestWorkflows.TestWorkflow1 workflowStub =
         testWorkflowRule
             .getWorkflowClient()
             .newWorkflowStub(
-                WorkflowTest.TestWorkflow1.class,
-                WorkflowTest.newWorkflowOptionsBuilder(testWorkflowRule.getTaskQueue()).build());
+                TestWorkflows.TestWorkflow1.class,
+                TestOptions.newWorkflowOptionsBuilder(testWorkflowRule.getTaskQueue()).build());
```

Comment:
Consider adding `testWorklfowRule.newWorkflowStub(<class>)` that sets options to the task queue from `testWorkflowRule.` So the above code becomes:
```java
TestWorkflows.TestWorkflow1 workflowStub = testWorkflowRule.newWorklfowStub(TestWorkflows.TestWorkflow1.class);
```